src/Data/Recruit.ChangeScripts/090-vacancyReviews_removing_employerAccountId.js

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Since there is a migration planned in the next release (from `vacancyReviews` to `reviews`) we need to make sure in which order this script gets executed.

Reply:
This file is also included in the PR for ER-887